src/orchard/inspect.clj

```diff
@@ -15,6 +15,20 @@
    (java.util List Map)
    clojure.lang.Seqable))
 
+;; Datafy and Nav are only available since Clojure 1.10
+
+(def ^:private datafy-supported?
+  "Whether Datafy and Nav are supported or not."
+  (some? (resolve 'clojure.core.protocols/datafy)))
+
+(defn- datafy [o]
+  (when datafy-supported?
+    ((resolve 'clojure.core.protocols/datafy) o)))
```

Comment:
Don't `resolve` each time, please do it once at compile-time

Reply:
Fixed